qos/evm/response_chainid.go

```diff
@@ -73,13 +71,11 @@ type responseToChainID struct {
 	// result captures the `result` field of a JSONRPC response to an `eth_chainId` request.
 	result string
 
-	// valid is set to true if the parsed response is deemed valid.
+	// Why the response has failed validation.
+	// Only set if the response is invalid.
 	// As of PR #152, a response is valid if either of the following holds:
```

Comment:
Consider updating #152 IFF applicable. Ditto elsewhere.

Reply:
No changes were made to the response validation conditions (responseNone introduced in this PR was always an invalid response)